client/my-sites/domains/domain-management/list/add-domain-button.jsx

```diff
@@ -49,35 +54,64 @@ class AddDomainButton extends React.Component {
 		this.props.trackAddDomainMenuClick( reactEvent.target.pathname );
 	};
 
+	renderOptions = () => {
+		const { specificSiteActions, translate } = this.props;
+
+		if ( specificSiteActions ) {
+			const useYourDomainUrl = domainUseYourDomain( this.props.selectedSiteSlug );
+			return (
+				<React.Fragment>
+					<PopoverMenuItem onClick={ this.clickAddDomain }>
+						{ translate( 'Search for a domain' ) }
+					</PopoverMenuItem>
+					<PopoverMenuItem href={ useYourDomainUrl } onClick={ this.trackMenuClick }>
+						{ translate( 'Use a domain I own' ) }
+					</PopoverMenuItem>
+				</React.Fragment>
+			);
+		}
+
+		return (
+			<React.Fragment>
+				{ this.props.selectedSiteSlug && (
+					<PopoverMenuItem onClick={ this.clickAddDomain }>
+						{ translate( 'to this site' ) }
+					</PopoverMenuItem>
+				) }
+				<PopoverMenuItem href="/new" onClick={ this.trackMenuClick }>
+					{ translate( 'to a new site' ) }
+				</PopoverMenuItem>
+				<PopoverMenuItem href="/domains/add" onClick={ this.trackMenuClick }>
+					{ translate( 'to a different site' ) }
+				</PopoverMenuItem>
+				<PopoverMenuItem href="/start/domain" onClick={ this.trackMenuClick }>
+					{ translate( 'without a site' ) }
+				</PopoverMenuItem>
+			</React.Fragment>
+		);
+	};
+
 	render() {
 		const { translate } = this.props;
 
+		const label = this.props.specificSiteActions
+			? translate( 'Add a domain to this site' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Add a domain to a new site' )` *ES Score: 8*